ui/src/utils/plugins.js

```diff
@@ -484,6 +484,14 @@ export const genericUtilPlugin = {
       const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi
       return regexExp.test(uuid)
     }
+
+    app.config.globalProperties.$toBase64Encoded = function (text) {
+      const base64regex = /^([0-9a-zA-Z+/]{4})*(([0-9a-zA-Z+/]{2}==)|([0-9a-zA-Z+/]{3}=))?$/
+      if (base64regex.test(text)) {
+        return text
```

Comment:
@shwstppr 
why not return `encodeURIComponent(text)` here ?

Reply:
@weizhouapache refactored

Reply:
@shwstppr 
I will verify it